app/controllers/application_controller.rb

```diff
@@ -36,7 +36,8 @@ class ApplicationController < ActionController::Base
   helper_method :current_templates
   helper_method :admin?, :logged_in?, :highlight_all
 
-  #  protect_from_forgery :secret => '112141be0ba20082c17b05c78c63f357'
+  protect_from_forgery with: :exception
```

Comment:
Are you referring to EmailsController ? Looking at its tests and test.logs after running tests, I think its working fine. But let me still revisit it, just to be sure... Specially those controller which don't have proper test coverage.